### ISSUE TITLE
Lumina pipeline fails to generate any image

### ISSUE THREAD

**Issue by zhuole1025** [huggingface/diffusers#9488]
### Describe the bug

I found the current version of diffusers fails to generate any image using Lumina pipeline. However, the 0.30.0 version works well. So I guess some related modules are changed during the update, but I have no idea after some debugging.

### Reproduction
```python
import torch
from diffusers import LuminaText2ImgPipeline

pipe = LuminaText2ImgPipeline.from_pretrained("Alpha-VLLM/Lumina-Next-SFT-diffusers", torch_dtype=torch.bfloat16).to("cuda")

img = pipe('a dog').images
```

### System Info

diffusers 0.31.0.dev0

**Comment by asomoza** [comment]
HI, I tested it with the current version and it works without a problem. 

We need more information about what happens to you, when you say "fails to generate" what does this mean? It just stops? throws a black image? or something else?

**Comment by zhuole1025** [comment]
Here is the generate images:
![image](https://github.com/user-attachments/assets/51c67832-7e9a-4fd7-b514-6341c9907f2d)

**Comment by asomoza** [comment]
Never seen that kind of failed image, I don't think there was any change to lumina between those 0.30 and the current one.

Kind of obvious question but did you try installing from scratch? I mean creating the whole environment again? As I answered before, I tested it and I can generate normal images with the current version.

If it still fails and need help, you'll need to post the details of your environment with the cli, as it says when you create the issue.
